### PR TITLE
Show warning when enabling proposed api for non-installed extensions

### LINKED ISSUE #55516: When running with --enable-proposed-api, no errors are shown in cli when a wrong extension name is provided
When running code from cli, below flags need extension id
- install-extension
- uninstall-extension
- enable-proposed-api

The first 2, show appropriate messages in then cli if given extension is not found
The last one doesnt

### PR DESCRIPTION
**Pull request by JacksonKearl**
This closes #55516 by showing a notification and logging to console when attempting to enable proposed api for an extension which is not installed.

### CODE REVIEW

**Comment by JacksonKearl** [comment]
ping @ramya-rao-a

**Review by ramya-rao-a — CHANGES_REQUESTED** [review]
For the sake of consistency with other validations in the same area in `_updateEnableProposedApi` method:

- Keep the id check case sensitive
- Use console.error instead of console.warn+notification

**Comment by ramya-rao-a** [comment]
Also keep the messaging consistent:

``Extension '${extension.id} cannot use PROPOSED API as it cannot be found` ?

File: src/vs/workbench/services/extensions/electron-browser/extensionService.ts [hunk 1]
```diff
@@ -551,6 +551,18 @@ export class ExtensionService extends Disposable implements IExtensionService {
 
 				const enableProposedApiFor: string | string[] = this._environmentService.args['enable-proposed-api'] || [];
 
+				const notFound = (id: string) => nls.localize('notFound', "Extension `${id}` cannot use PROPOSED API as it cannot be found", id);
+
+				if (enableProposedApiFor.length) {
+					let allProposed = (enableProposedApiFor instanceof Array ? enableProposedApiFor : [enableProposedApiFor]);
+					let allInstalled = allExtensions.map(description => description.id);
```
Comment (ramya-rao-a) [inline src/vs/workbench/services/extensions/electron-browser/extensionService.ts]: Can we not skip the map and use the `allExtensions.some()` array directly?

**Review by ramya-rao-a — APPROVED** [review]

**Pull request merged**